src/decision/location_selection.jl

```diff
@@ -9,211 +9,245 @@ using ADRIA:
     n_locations,
     to_coral_spec,
     colony_mean_area,
-    switch_RCPs!
+    switch_RCPs!,
+    DataCube,
+    copy_datacube
 
 
 """
-    rank_locations(domain::Domain, scenarios::DataFrame, area_to_seed::Float64; target_seed_sites=nothing, target_fog_sites=nothing)::NamedDimsArray
-    rank_locations(domain::Domain,scenarios::DataFrame, area_to_seed::Float64, agg_func::Function,
-        iv_type::Union{String,Int64}; target_seed_sites=nothing, target_fog_sites=nothing)::AbstractArray
-
-Return location ranks for a given domain and scenarios.
+    rank_locations(
+        dom::Domain,
+        n_corals::Int64,
+        scenarios::DataFrame;
+        rcp=nothing,
+        min_iv_locs=nothing,
+        max_members=nothing,
+        target_seed_locs=nothing,
+        target_fog_locs=nothing
+    )::YAXArray
+
+Determine location ranks for a given domain and intervention scenarios.
+Locations are ranked by order of their determined deployment suitability according to
+criteria weights. Values of 1 indicate highest rank.
+
+Interventions are assessed separately such that seeding locations are not influenced by
```

Comment:
Are we removing the case where we use the same set of sites to fog where sites were seeded?